lib/fuse.c

```diff
@@ -2616,6 +2618,12 @@ static void fuse_lib_init(void *data, struct fuse_conn_info *conn)
 	/* Disable the receiving and processing of FUSE_INTERRUPT requests */
 	if (!f->conf.intr)
 		conn->no_interrupt = 1;
+
+	if (f->conf.intr &&
+	    fuse_init_intr_signal(f->conf.intr_signal,
+			          &f->intr_installed) == -1) {
+		fuse_log(FUSE_LOG_ERR, "fuse: failed to init interrupt signal\n");
+	}
```

Comment:
Maybe rewrite this a little bit and merge the `f->conf.intr` conditions?
```
if (f->conf.intr) {
    if ( fuse_init_intr_signal(f->conf.intr_signal,
			          &f->intr_installed) == -1)
        fuse_log(FUSE_LOG_ERR, "fuse: failed to init interrupt signal\n");
} else {
    conn->no_interrupt = 1;
}
```

Reply:
Thanks for your advice！The code is updated.